### DISCUSSION TITLE
Sykz's submission (chapter1)

### DISCUSSION THREAD

**Discussion by Sykzpixel** [stellar/sorobanathon#95] [General]
https://litemint.com/items/GDQXAMVRFB3INTXE3LXK33VA3LAS77C4MOGVMON4GANSMFJUW7YFNWDR/sykz1

**Comment by kalepail** [comment]
This is absolutely unreal 🤯

**Reply by rawritude** [reply]
You should get verified on litemint. This would be an awesome featured collection. Welcome to Stellar!!!!

**Comment by stellarjohn** [comment]
Hi Sykz,

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
